deploy/sdk/src/dynamo/sdk/request_tracing.py

```
@@ -0,0 +1,370 @@
+                        adj_idx = param_idx - 1  # skip `self`
+                        if 0 <= adj_idx < len(args):
+                            request_id = args[adj_idx]
+
+                            # Remove the positional argument and convert remaining args to kwargs
+                            # to avoid position mismatches
+                            remaining_args = (
+                                *args[:adj_idx],
+                                *args[adj_idx + 1 :],
+                            )
+
+                            # Convert remaining positional args to keyword args using function signature
```

Pylint (3.3.7)

[convention] Line too long (104/100)

(C0301)